Added AKS blog post (#27)
paralus/website: blog/2022-06-30-paralus-quickstart-aks/index.md

After the cluster is created, start the cluster and connect to it. If you've created a cluster using the Azure portal, you can use the Azure Cloud Shell to connect to the cluster.

## Installing Paralus

1. Add helm repo

   `helm repo add paralus https://paralus.github.io/helm-charts`

2. Install Paralus

  ```bash
   helm install myrelease paralus/ztka \
    -f https://raw.githubusercontent.com/paralus/helm-charts/main/examples/values.aks.yaml \
    -set fqdn.domain="chartexample.com" \
    -n paralus \
    --create-namespace
  ```

  >Note: If you plan to use Paralus without a `https` enabled domain, you'll need to set the `kratos.development` flas as `true`

  ```bash
   NAME: myrelease
   LAST DEPLOYED: Wed Jun 29 10:13:48 2022
   NAMESPACE: paralus
   STATUS: deployed
   REVISION: 1
   NOTES:
   1. Access the application URL by running these commands:
     Get load balancer address via:
     kubectl get service envoy --namespace paralus -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'

     Add DNS records of following domains such that it resolves to above address:

>      - console-aks-oss.chartexample.com

Reviewer: Since we are not updating subdomains, Update these domains to
- console.chartexample.com
- *.core-connector.chartexample.com
- *.user.chartexample.com

Also, same change in DNS setting section below.

Author: Fixed

     - *.core-connector.aks-oss.chartexample.com
     - *.user.aks-oss.chartexample.com

>      Open http://console-aks-oss.chartexample.com in browser.

Reviewer: ```suggestion
     Open http://console.chartexample.com in browser.
```

elasticsearch-master-headless   ClusterIP      None           <none>         9200/TCP,9300/TCP               3m13s
myrelease-contour               ClusterIP      10.0.151.61    <none>         8001/TCP                        3m13s
myrelease-contour-envoy         LoadBalancer   10.0.33.6      13.71.51.105   80:30193/TCP,443:30873/TCP      3m13s
myrelease-kratos-admin          ClusterIP      10.0.16.90     <none>         80/TCP                          3m13s
myrelease-kratos-courier        ClusterIP      None           <none>         80/TCP                          3m13s
myrelease-kratos-public         ClusterIP      10.0.234.235   <none>         80/TCP                          3m13s
myrelease-postgresql            ClusterIP      10.0.114.76    <none>         5432/TCP                        3m13s
myrelease-postgresql-hl         ClusterIP      None           <none>         5432/TCP                        3m13s
paralus                         ClusterIP      10.0.12.103    <none>         11000/TCP,10000/TCP,10001/TCP   3m13s
prompt                          ClusterIP      10.0.50.46     <none>         7009/TCP                        3m13s
relay-server      
```

> Note down the IP address for the service type `LoadBalancer`

Reviewer 2: Should we be explicit about it being the `externall-ip` of `<releasename>-contour-envoy`?

Author: In case of AWS, we did get a [fqdn instead of external IP](https://www.paralus.io/blog/eks-quickstart#configuring-dns-settings), however in case of Azure got the IP address which should be configured in DNS settings.

Reviewer 2: What I meant was we could change it  to be explicit instead of just saying get the ip of the LB.
```suggestion
Note down the `EXTERNAL-IP` address for the `<releasename>-contour-envoy` service.
```

| Type | Address | Resolves To | TTL |
|---|---|---|---|
| A | console-aks-oss.chartexample.com | 13.71.51.105 | 1 Hour |
| A | *.core-connector.aks-oss.chartexample.com | 13.71.51.105 | 1 Hour |
| A | *.user.aks-oss.chartexample.com | 13.71.51.105 | 1 Hour |

### Accessing The Dashboard

Paralus is installed with a default organization and an admin user. Hence, after installation, you need to set a password for the user. To do so, execute the command that you get after installing Paralus.

```bash
kubectl logs -f --namespace paralus $(kubectl get pods --namespace paralus -l app.kubernetes.io/name='paralus' -o jsonpath='{ .items[0].metadata.name }') initialize | grep 'Org Admin signup URL:'

Org Admin signup URL:  http://console.chartexample.com/self-service/recovery?flow=de34efa4-934e-4916-8d3f-a1c6ce65ba39&token=IYJFI5vbORhGnz81gCjK7kucDVoiuQ7j

```

Access the URL in a browser, and provide a new password. In a new browser window/tab navigate to `http://console.chartexample.com` and log in with the following credentials:

> - username: `foo@example.com`

Reviewer 2: It will be the the value they specific in the `values.yaml` file and by default it is `admin@paralus.local`.

Author: Got it.

---
slug: paralus-quickstart-aks

> title: "Tutorial: How to set up Paralus on Azure Kubernetes Service - AKS"

Reviewer: ```suggestion
title: "Tutorial: How to set up Paralus on Azure Kubernetes Service (AKS)"
```

Author: Fixed.

## Creating AKS Cluster

One of the good things when it comes to creating an AKS cluster is the various options it gives you to create a cluster:

- [Using the Azure CLI](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli)
- [Using the Azure PowerShell](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-powershell)
- [Using the Azure Portal](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-powershell)
- [Using the ARM template](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-rm-template)

You are free to choose any option that you want, we used the Azure Portal to create one.

> Note: Choose a virtual machine with atleast 4 vCPUs and 16 GB of RAM. Refer to the list of [Azure VMs](https://azure.microsoft.com/en-in/pricing/details/virtual-machines/linux/).

After the cluster is created, start the cluster and connect to it. If you've created a cluster using the Azure portal, you can use the Azure Cloud Shell to connect to the cluster.

## Installing Paralus

> Clone the [paralus helm repository](https://github.com/paralus/helm-charts)

Reviewer: This step is not required.

Author: Removed from this as well a EKS blog post.

1. Add helm repo

   `helm repo add paralus https://paralus.github.io/helm-charts`

2. Create `values.aks.yaml`

   ```yaml
   ingress:

>      enabled: false

Reviewer: default is false. So this is not required.

Author: Removed from this as well a EKS blog post.

   REVISION: 1
   NOTES:
   1. Access the application URL by running these commands:
     Get load balancer address via:
     kubectl get service envoy --namespace paralus -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'

     Add DNS records of following domains such that it resolves to above address:
     - console-aks-oss.chartexample.com
     - *.core-connector.aks-oss.chartexample.com
     - *.user.aks-oss.chartexample.com
     Open http://console-aks-oss.chartexample.com in browser.

   You can view the recovery link for admin user by running the following command once all the pods are running:

   kubectl logs -f --namespace paralus $(kubectl get pods --namespace paralus -l app.kubernetes.io/name='paralus' -o jsonpath='{ .items[0].metadata.name }') initialize-paralus | grep 'Org Admin signup URL:'
   ```

   > Note: It can take upto a few minutes before all the pods are running and you can access the dashboard. You can check the status using `watch kubectl get pods`

## Configuring DNS Settings

Once the installation is complete, you need to first get the external IP address provided by Azure loadbalancer. You can do so by executing the following command:

```bash

> kubectl get svc -n paralus

Reviewer: `kubectl get svc myrelease-contour-envoy -n paralus` is more specific command. what do you think?

Author: Done

   `helm repo add paralus https://paralus.github.io/helm-charts`

2. Install Paralus

  ```bash
   helm install myrelease paralus/ztka \
    -f https://raw.githubusercontent.com/paralus/helm-charts/main/examples/values.aks.yaml \
    -set fqdn.domain="chartexample.com" \
    -n paralus \
    --create-namespace
  ```

>   >Note: If you plan to use Paralus without a `https` enabled domain, you'll need to set the `kratos.development` flas as `true`

Reviewer 2: ```suggestion
  >Note: If you plan to use Paralus without a `https` enabled domain, you'll need to set `kratos.development` in `values.yaml` as `true`
```
